user_guide_src/source/cli/cli_generators.rst

```diff
@@ -308,3 +308,14 @@ GeneratorTrait
 
 All generator commands must use the ``GeneratorTrait`` to fully utilize its methods that are used in code
 generation.
+
+*************************************
+Accessing a Custom Generator Template
+*************************************
+
+By default, all generator templates will be looked up at the ``CodeIgniter\Commands\Generators\Views`` namespace.
+To declare a custom location for your custom generator template, you will need to add it to the **app/Config/Generators.php**
+file. For example, if you have a command ``make:awesome-command`` and your generator template is located within your *app*
+directory **app/Commands/Generators/Views/awesomecommand.tpl.php**, you would update the config file like so:
```

Comment:
This is wrong. The order of lookup is:
1) The template defined in `app/Config/Generators.php`;
2) The template in the `CodeIgniter\Commands\Generators\Views` namespace.

https://github.com/codeigniter4/CodeIgniter4/blob/848c59860b40355f0de10c1dd9ad3b4761e39b2c/system/CLI/GeneratorTrait.php#L267-L276

Reply:
Thanks for point that out.